### DISCUSSION TITLE
How to I upgrade the react native version if I used this tool previously?

### DISCUSSION THREAD

**Discussion by ThakurKarthik** [microsoft/rnx-kit#3418] [Q&A]
Hi I upgraded to react-native 0.74.0 previously using this tool. My package.json has a section
![image](https://github.com/user-attachments/assets/02c0d844-305b-48d9-befe-bf6c603cbcc0)

Now I am trying to upgrade to 0.76.0 but the tool still showing the 0.74 deps only, After looking at this url https://react-native-community.github.io/upgrade-helper/?from=0.74.2&to=0.76.2&package=com.myfinancebuddy&name=MyFinanceBuddy
I ran this command 
`npx @rnx-kit/align-deps --requirements=react-native@0.76 --verbose`

but it still shows the 0.74 deps, please see below screenshot of the terminal.
![image](https://github.com/user-attachments/assets/3f21e487-8fc9-438b-9d31-b4740e76c9d9)
I also tried to install latest version of the toolkit "@rnx-kit/align-deps": "^3.0.2" and ran 
![image](https://github.com/user-attachments/assets/8be5a50b-5058-4eca-8c39-6f559264e7cc)
but still same issue, I think I am missing something simple here, please help and thanks in advance!

**Comment by Saadnajmi** [comment]
Just to check, if you change the entry in all your package.json files to "react-native@0.76", and then run "align-deps --write" that doesn't work?

**Reply by ThakurKarthik** [reply]
I haven't tried that. I was thinking the cli command will take care of it automatically

**Comment by tido64** [comment]
Hey, if you take a look in the docs, `--requirements` is only applied to **_unconfigured_** packages. What you want to use is `--set-version`. You can read more about it here: https://microsoft.github.io/rnx-kit/docs/tools/align-deps#--set-version

As [mentioned above](https://github.com/microsoft/rnx-kit/discussions/3418#discussioncomment-11285069), an alternative is to manually change `react-native@0.74` → `react-native@0.76` and re-run the tool.
